web-ui/src/components/Profile/profile.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import "./styles.css";
+
+class Profile extends Component{
+    render() {
+        return (
+            <div className="card UserProfile">
+                <img src="https://www.bsn.eu/wp-content/uploads/2016/12/user-icon-image-placeholder-300-grey.jpg" alt="User Placeholder" className="img-thumbnail rounded-circle"/>
```

Comment:
I think it's better to download the images and reference locally.

Reply:
I agree with @bhushanmalgaonkar

Reply:
I'll fix it in next commit